app/views/directives/header/header.html

```diff
@@ -27,17 +27,20 @@
     <!-- Use title="" to prevent "Nothing selected" from briefly flashing while the projects are still loading. -->
     <select class="selectpicker form-control" data-selected-text-format="count>3" id="boostrapSelect" title=""></select>
   </div>
+  <catalog-search ng-if="canIAddToProject" catalog-items="catalogItems" base-project-url="project" toggle-at-mobile="true" search-toggle-callback="onSearchToggle"></catalog-search>
   <!-- create buttons show at mobile -->
   <div class="dropdown add-to-project" ng-if="canIAddToProject" uib-dropdown>
-    <a class="dropdown-toggle" href="" ng-disabled="currentProject.status.phase != 'Active'" title="Add to Project" uib-dropdown-toggle>
-      <i class="fa fa-plus visible-xs-inline-block" aria-hidden="true" title="Add to Project"></i><span class="hidden-xs">Add to Project</span><span class="hidden-xs caret" aria-hidden="true" title="Add to Project"></span>
-    </a>
+    <button class="dropdown-toggle btn btn-link" ng-disabled="currentProject.status.phase != 'Active'" title="Add to Project" uib-dropdown-toggle>
+      <i class="fa fa-plus visible-xs-inline-block" aria-hidden="true" title="Add to Project"></i>
+      <span class="hidden-xs">Add to Project</span>
+      <span class="hidden-xs caret" aria-hidden="true" title="Add to Project"></span>
+    </button>
     <ul role="menu" uib-dropdown-menu class="dropdown-menu dropdown-menu-right">
       <li ng-if-start="!catalogLandingPageEnabled" role="menuitem"><a ng-href="project/{{currentProjectName}}/create?tab=fromCatalog">Browse Catalog</a></li>
       <li role="menuitem"><a ng-href="project/{{currentProjectName}}/create?tab=deployImage">Deploy Image</a></li>
       <li ng-if-end role="menuitem"><a ng-href="project/{{currentProjectName}}/create?tab=fromFile">Import YAML / JSON</a></li>
 
-      <li ng-if-start="catalogLandingPageEnabled" role="menuitem"><a href="./">Browse Catalog</a></li>
+      <li ng-if-start="catalogLandingPageEnabled" role="menuitem"><a href="project/{{currentProjectName}}/project-browse-catalog">Browse Catalog</a></li>
```

Comment:
We should wait on this until the in-project browse works, correct?

Reply:
yes. I will remove this change.